debezium-ddl-parser/src/main/antlr4/io/debezium/ddl/parser/mysql/generated/MySqlParser.g4

```diff
@@ -609,6 +609,7 @@ alterSpecification
     | CHANGE COLUMN? oldColumn=uid
       newColumn=uid columnDefinition 
       (FIRST | AFTER afterColumn=uid)?                              #alterByChangeColumn
+    | RENAME COLUMN? oldColumn=uid TO newColumn=uid                 #alterByRenameColumn
```

Comment:
Is the `COLUMN` token really optional in the grammar?

Reply:
yeah it isn't. Removed that and created a new PR.